### ISSUE TITLE
📝 Correction | Put back Matrix and XMPP again

### ISSUE THREAD

**Issue by Kcchouette** [privacytoolsIO/privacytools.io#1854]
## Description

The "Cleanup 2.0 instant messenger page" (https://github.com/privacytoolsIO/privacytools.io/commit/94629b75f76d50065a77cd41d41f198066f26a69) has removed all reference to XMPP section

## Why I am making the suggestion

1. Ok why not removing Tox because of issue https://github.com/TokTok/c-toxcore/issues/426
1. why changed Matrix to Riot? For eg the Riot android app listed on the website use the non-free vector software: https://f-droid.org/packages/im.vector.alpha/. Matrix is not only Riot, it's at the minimum https://matrix.to/#/#general:privacytools.io and https://matrix.org/docs/projects/try-matrix-now
1. XMPP has completely disappeared, even if 19 clients has implemented and added the possibility to use OMEMO E2E https://omemo.top/ (but another issue seems to include that: https://github.com/privacytoolsIO/privacytools.io/issues/1838 )

## My connection with the software

None

**Comment by dngray** [comment]
Please see the relevant discussion:

- https://github.com/privacytoolsIO/privacytools.io/pull/1500#discussion_r350414878
- https://github.com/privacytoolsIO/privacytools.io/pull/1836

TLDR is: Many of the XMPP clients are not stable enough and do not support E2EE on by default for private conversations (something which will [imminently happen with Riot](https://github.com/vector-im/riot-web/issues/6779)). Conversations is an exception, however there's no point in recommending one client for one platform.

XMPP clients do not use E2EE for VOIP related features. This is a "gotcha" that also influenced us removing our recommendation. See our tracking issue https://github.com/privacytoolsIO/privacytools.io/issues/1838#issuecomment-616025568

**Comment by Kcchouette** [comment]
Thanks for your response

**Comment by Relected** [comment]
how about blabber.im? it offers E2EE using OMEMO by default, and its based on conversation
